source/resume.html

```diff
@@ -0,0 +1,189 @@
+          <p>
+            <b>Women’s Center</b> Marquette, MI
+            <em>September 2017</em>
+            <br>Design and Develop
+            <br> The Women’s Center project was the largest project I’ve done and involved a full redesign, and content management system(CMS) and hosting transfer. I created the design and used Wordpress as the CMS to allow for client control and adjustments in the future. I worked directly with the client throughout the process to ensure that content and functionality were correct and that design fit the tone of their brand and mission.</p>
+          <p>
+            <b>Derouin’s Auto Shop</b> Marquette, MI
+            <em>April 2018</em>
+            <br>Design and Develop
+            <br> Derouin’s Auto Shop was my first paid job. The client was simply looking for a fresh look and to make the site mobile friendly. I created the design and built the site on Wordpress, allowing the client to make updates and perform common maintenance tasks.</p>
+        </div>
+      </div>
+      <div class="hobbies">
+        <h3>outside of work</h3>
+        <div class="nerd-stuff">
+          <ul>
+            <h4>hobbies:</h4>
+            <li>Wood Working</li>
+            <li>Beer Brewing</li>
+            <li>Beer Drinking</li>
+            <li>Photography</li>
+            <li>Learning Guitar</li>
+            <li>Lego Building</li>
+            <li>Reading</li>
+            <li>Dad Jokes</li>
+          </ul>
+        </div>
+      </div>
+    </section>
+    <footer>
+      <img src="./assets/pie.png">
+      <div class="foot-nav">
+        <div class="footer-menu">
+          <ul>
+            <li><h4>Services</h4></li>
+            <li>Web Design</li>
+            <li>Development</li>
+            <li>Graphic Design</li>
+            <li>Branding</li>
+          </ul>
+        </div>
+        <div class="footer-menu">
+          <ul>
+            <li><h4>Company</h4></li>
+            <li>
+              <a href="index.html#about">About</a>
+            </li>
+            <li>
+              <a href="resume.html">Resume</a>
+            </li>
+            <li>
+              <a href="index.html#projects">Projects</a>
+            </li>
+            <li>
+              <a href="index.html#contact">Contact</a>
+            </li>
+          </ul>
+        </div>
+        <div class="footer-menu">
+          <ul>
+            <li><h4>Links</h4></li>
+            <li>
+              <a href="https://www.facebook.com/" target="_blank">Facebook</a>
+            </li>
+            <li>
+              <a href="https://www.instagram.com/" target="_blank">Instagram</a>
+            </li>
+            <li>
+              <a href="https://www.linkedin.com/feed/" target="_blank">LinkedIn</a>
+            </li>
+          </ul>
+        </div>
+        <div class="footer-menu">
+          <ul>
+            <li><h4>Get in touch</h4></li>
+            <li><a href="mailto:mark.m.hayes1@gmail.com">Email@me</a></li>
```

Comment:
I'd be careful with something this cute … some idiot is going to try to send an email to `Email@me`.